<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmPesqAluguel</class>
 <widget class="QDialog" name="FrmPesqAluguel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lista de Aluguéis</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Imagens/btnListAluguel.png</normaloff>../../Imagens/btnListAluguel.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>501</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Selecione o Tipo de Pesquisa</string>
   </property>
   <widget class="QComboBox" name="cbPesquisa">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <item>
     <property name="text">
      <string>Código Aluguel</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Código Cliente</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Código Veículo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nome Cliente</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="edtPesquisa">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPesquisa">
    <property name="geometry">
     <rect>
      <x>404</x>
      <y>10</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Pesquisar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Imagens/lupa.png</normaloff>../../Imagens/lupa.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>501</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Código Aluguel</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nome Cliente</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Data Aluguel</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Data Prazo</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Data Devolução</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Valor Aluguel</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Valor Multa</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Km Entrada</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Km Saída</string>
    </property>
   </column>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>501</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Devolver Veículo</string>
   </property>
   <widget class="QPushButton" name="btnDevolver">
    <property name="geometry">
     <rect>
      <x>394</x>
      <y>10</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Devolver</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Imagens/return.png</normaloff>../../Imagens/return.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Data Devolução</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtDevolucao">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtMulta">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtSaida">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Multa</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Km Saída</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
